hack/install-wwhrd.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
Why the `hack` directory? Is it a temporary implementation?

Reply:
it is a habit that I took with the kubernetes repositories: https://github.com/kubernetes/kubernetes/tree/master/hack

Reply:
we can rename it

Reply:
I'm wondering if `tools/licenses` would not be an appropriate place for instance

Reply:
+1 to use the `tools` dir that we already have for this kind of scripts/helpers.

Reply:
agree

Reply:
done